### ISSUE TITLE
When exporting from command line, --no-window is not respected.

### ISSUE THREAD

**Issue by spacechase0** [godotengine/godot#43598]
**Godot version:**
3.2.3

**OS/device including version:**
Windows 10

**Issue description:**
Whenever I export from the command line, the window still pops up despite specifying `--no-window`. This happens with both `--export` and `--export-pack`.

**Steps to reproduce:**
The command I'm using is:
```
"G:\Godot_v3.2.3-stable_mono_win64\Godot_v3.2.3-stable_mono_win64.exe" --export "Windows Desktop" ../export/Test.exe --no-window
```

**Minimal reproduction project:**
[test.zip](https://github.com/godotengine/godot/files/5549207/test.zip)

**Comment by Xrayez** [comment]
I confirm this, I've stumbled upon this issue some time ago.

To note, `--no-window` is only implemented on Windows, so can only be reproduced on Windows.

**Comment by Calinou** [comment]
cc @RandomShaper as they implemented no-window mode on other platforms. They might know about this :slightly_smiling_face:

**Comment by akien-mga** [comment]
Fixed by #43631.
